src/services/exports/controllers/getExportConseillersCsv.ts

```diff
@@ -8,12 +9,86 @@ import {
   filterNomConseiller,
   filterRegion,
   filterNomStructure,
-  filterIsRupture,
+  filterIsRuptureConseiller,
+  filterIsRuptureMisesEnRelation,
+  checkAccessReadRequestConseillers,
 } from '../../conseillers/conseillers.repository';
 import { generateCsvConseillers } from '../exports.repository';
 import { getNombreCras } from '../../cras/cras.repository';
 import checkAccessReadRequestMisesEnRelation from '../../misesEnRelation/misesEnRelation.repository';
 
+const getConseillersRecruter =
+  (app: Application, checkAccess) =>
+  async (
+    dateDebut: Date,
+    dateFin: Date,
+    isCoordinateur: string,
+    searchByConseiller: string,
+    region: string,
+    rupture: string,
+  ) =>
+    app.service(service.conseillers).Model.aggregate([
+      {
+        $match: {
+          ...filterIsCoordinateur(isCoordinateur),
+          ...filterNomConseiller(searchByConseiller),
+          ...filterRegion(region),
+          ...filterIsRuptureConseiller(rupture, dateDebut, dateFin),
+          $and: [checkAccess],
+        },
+      },
+      {
+        $project: {
+          _id: 1,
+          structureId: 1,
+        },
+      },
+    ]);
+
+const getMisesEnRelationRecruter =
+  (app: Application, checkAccess) =>
+  async (
+    rupture: string,
+    searchByStructure: string,
+    structureIds: ObjectId[],
+    conseillerIds: ObjectId[],
+    sortColonne: object,
+  ) =>
+    app.service(service.misesEnRelation).Model.aggregate([
+      {
+        $match: {
+          ...filterIsRuptureMisesEnRelation(
+            rupture,
+            conseillerIds,
+            structureIds,
+          ),
+          ...filterNomStructure(searchByStructure),
+          $and: [checkAccess],
+        },
+      },
+      {
+        $project: {
+          _id: 0,
+          'conseillerObj._id': 1,
+          'conseillerObj.idPG': 1,
+          'conseillerObj.prenom': 1,
+          'conseillerObj.nom': 1,
+          'conseillerObj.emailCN.address': 1,
```

Comment:
normalement ca devrait pas arriver

```suggestion
          'conseillerObj.emailCN?.address': 1,
```